.travis.yml

```diff
@@ -77,7 +77,7 @@ install:
   - gcc --version
   - |
     if [ -n "${XXHASH_BUILDWHEELS:-}" ]; then
-      pip install cibuildwheel
+      pip install cibuildwheel==1.7.0
```

Comment:
I think in some tests pip is actually from python 2, we'd like to make sure that it is pip3.
Maybe we can try `python3 -m pip install cibuildwheel`

Reply:
Sure, updated.